### ISSUE TITLE
sample-sheets that have positive controls - swe need to know what it is exactly 

### ISSUE THREAD

**Issue by charles-cowart** [biocore/metagenomics_pooling_notebook#238]
Suggestion raised in Futuretech meeting: For our sample-sheets that have positive controls, we know exactly what they reference. Not sure I captured the request in its entirely - @AmandaBirmingham what do you think?

**Comment by AmandaBirmingham** [comment]
I think this was referring to the need to be able to tell from a sample sheet things like what particular genome was used for katharoseq positive controls.

**Comment by wasade** [comment]
Yes although ideally for any positive control, not just KatharoSeq
